packages/paste-website/src/pages/form-elements/select/index.mdx

```diff
@@ -0,0 +1,943 @@
+    Make sure to connect the <inlineCode>FormLabel</inlineCode> to the <inlineCode>Select</inlineCode>. This is done with the <inlineCode>htmlFor</inlineCode> prop on the label, and the <inlineCode>id</inlineCode> prop on the select. Those two need to match.
+  </CalloutText>
+</Callout>
+
+<LivePreview scope={{Select, Option, FormHelpText, FormLabel}} language="jsx">
+  {`<>
+  <FormLabel htmlFor="select_product_demo" required>Select a Product</FormLabel>
+  <Select id="select_product_demo" htmlFor="select_product_demo" required>
+    <Option value="messaging">SMS</Option>
+    <Option value="numbers">Phone Numbers</Option>
+    <Option value="video">Video</Option>
+  </Select>
+  <FormHelpText variant="default" id="select_product">Please choose a Twilio product.</FormHelpText>
+</>`}
+</LivePreview>
+
+### Select with add-ons (prefix/suffix text or icons)
+
+- **Prefix/suffix text** - Text that can be used as a prefix or suffix to the value that is entered. Use prefix/suffix to help users format text.
+- **Icon** - Icons that can be placed in the same area as the prefix and suffix text. Icons should trigger an action (*e.g.*, opening a popover).
+
+<LivePreview scope={{Select, Option, FormHelpText, FormLabel}} language="jsx">
+  {`<>
+  <FormLabel htmlFor="select_add_ons_prefix_demo">Phone Number</FormLabel>
+  <Select
+    id="select_add_ons_prefix_demo"
+    htmlFor="select_add_ons_prefix_demo"
+    insertBefore={<div>+1</div>}
+  >
+    <Option value="cats">(415) 555-CATS</Option>
+    <Option value="dogs">(415) 555-DOGS</Option>
+    <Option value="mice">(415) 555-MICE</Option>
+  </Select>
+</>`}
+</LivePreview>
+
+<LivePreview scope={{Select, Option, FormHelpText, FormLabel, InformationIcon, Button}} language="jsx">
+  {`<>
+  <FormLabel htmlFor="select_add_ons_suffix_demo">Phone Number</FormLabel>
+  <Select
+    id="select_add_ons_suffix_demo"
+    htmlFor="select_add_ons_suffix_demo"
+    insertAfter={
+      <Button variant="link">
+        <InformationIcon decorative={false} title="Get more info" />
+      </Button>
+    }
+  >
+    <Option value="cats">(415) 555-CATS</Option>
+    <Option value="dogs">(415) 555-DOGS</Option>
+    <Option value="mice">(415) 555-MICE</Option>
+  </Select>
+</>`}
+</LivePreview>
+
+### Select with default values
+
+To be consistent with <Anchor target="_blank" href="https://reactjs.org/docs/uncontrolled-components.html#default-values">React's documentation</Anchor> on best practices for native select elements, the <inlineCode>selected</inlineCode> attribute is not allowed in the API. 
+You can instead select a <inlineCode>defaultValue</inlineCode> and supply that on render as the <inlineCode>value</inlineCode> property for the select.
+
+<LivePreview scope={{Select, Option, FormHelpText, FormLabel}} language="jsx">
+  {`<>
+  <FormLabel htmlFor="default_value_demo" required>Phone Number</FormLabel>
+  <Select
+    id="default_value_demo"
+    htmlFor="default_value_demo"
+    value="dogs"
```

Comment:
Should this be `defaultValue='dogs'`?

Reply:
I think the docs are just a leftover, we’re encouraging folks to use value, but defaultValue is available under the hood. Just need the docs to suggest value over defaultValue and the confusion hopefully is resolved

Reply:
Perhaps this is a case where I should write a component `<MySelect />`, pass it a default value and show some logic. In this case, the `defaultValue` is not an accepted prop on the Select interface, but the suggestion here is to use a `defualtValue` on their interface and pass that to `Select`. Something like, on the select component `value={defaultValue || value}`